### PR TITLE
MINOR : supported rtl in sidebar

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

- I worked on supporting `RTL` in sidebar

https://github.com/open-metadata/OpenMetadata/assets/66266464/fc38e745-dfe8-4786-9388-058243372c1a

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/AppContainer/AppContainer.tsx [hunk 1]
```diff
@@ -23,17 +25,25 @@ import LeftSidebar from '../MyData/LeftSidebar/LeftSidebar.component';
 import './app-container.less';
 
 const AppContainer = () => {
+  const { i18n } = useTranslation();
+  const direction = i18n.dir();
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/AppContainer/AppContainer.tsx:29]:
```suggestion
```

File: openmetadata-ui/src/main/resources/ui/src/components/AppContainer/AppContainer.tsx [hunk 2]
```diff
@@ -23,17 +25,25 @@ import LeftSidebar from '../MyData/LeftSidebar/LeftSidebar.component';
...
   const { Header, Sider, Content } = Layout;
   const { currentUser } = useAuthContext();
 
+  const isDirectionRTL = useMemo(() => direction === 'rtl', [direction]);
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/AppContainer/AppContainer.tsx:33]:
```suggestion
  const isDirectionRTL = useMemo(() => i18n.dir() === 'rtl', [i18n]);
```

File: openmetadata-ui/src/main/resources/ui/src/components/MyData/LeftSidebar/left-sidebar.less [hunk 3]
```diff
@@ -271,3 +271,50 @@
...
+  }
+
+  .brand-logo-container {
+    padding-left: initial;
+    padding-right: 14px;
+  }
+
+  .left-sidebar-menu.ant-menu {
+    .ant-menu-item {
+      padding: 8px 13px 8px 0 !important;
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/MyData/LeftSidebar/left-sidebar.less:290]: Can we avoid `!important`?

**Comment by Ashish8689** [comment]
@Sachin-chaurasiya  changes made as per comments!

File: openmetadata-ui/src/main/resources/ui/src/components/MyData/LeftSidebar/left-sidebar.less (hunk 3 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/MyData/LeftSidebar/left-sidebar.less:290]: antd items using inline styles for padding and we can only override them using important here and other places

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**